util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
>OS isn't coming from the environment. It's defined by the makefile on line 81. :)

Did you make sure? See my screenshot. If you re-define in Makefile, it's not right way, I think.